tool/tsh/tsh.go

```diff
@@ -270,6 +274,7 @@ func Run(args []string) {
 	ls.Flag("cluster", clusterHelp).Envar(clusterEnvVar).StringVar(&cf.SiteName)
 	ls.Arg("labels", "List of labels to filter node list").StringVar(&cf.UserHost)
 	ls.Flag("verbose", "One-line output, including node UUIDs").Short('v').BoolVar(&cf.Verbose)
+	ls.Flag("output", "Format output (json, text)").Default(teleport.Table).StringVar(&cf.OutputFormat)
```

Comment:
This flag should be named `format` in order to be consistent with the rest of our CLIs ([example](https://github.com/gravitational/teleport/blob/83060829a33d7fa242a9277aa1f8d3d182baa634/tool/tctl/common/resource_command.go#L101)).